src/main/java/au/com/codeka/carrot/Bindings.java

```diff
@@ -0,0 +1,13 @@
+package au.com.codeka.carrot;
+
+/**
+ * The Bindings of a template when rendered.
+ *
+ * @author marten
+ */
+public interface Bindings
+{
```

Comment:
that's an absolutely reasonable request. In our projects we use a different code style and I'm just used to that.
If you use IntelliJ it might make sense to put your project code style settings under version control.